packages/elements-core/src/components/TryIt/Parameters/parameter-utils.ts

```diff
@@ -81,10 +81,8 @@ const getInitialValueForParameter = (parameter: ParameterSpec) => {
 };
 
 export const initialParameterValues: (params: readonly ParameterSpec[]) => Record<string, string> = params => {
-  return _.chain(params)
```

Comment:
Chain cannot be tree-shaken, and thus the rollup lodash plugin that re-writes lodash imports for bundle size + esm compat does not work on it. We should avoid using chain.